src/components/long-number.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { formatLargeNumber } from '../utils';
+
+const LongNumber = ({ number }) => <Fragment>{formatLargeNumber(number)}</Fragment>;
```

Comment:
As there is only one params into the component I am wondering if it would be better to have this a child? Ie
`<LongNumber>1000000</LongNumber>`?